bc_obps/service/data_access_service/user_service.py

```diff
@@ -74,3 +74,8 @@ def update_user(cls, user_guid: UUID, updated_data: UserUpdateIn) -> User:
             setattr(user, attr, value)
         user.save()
         return user
+
+    @classmethod
+    def user_has_access_to_operator(cls, user_guid: UUID, operator_id: UUID) -> bool:
```

Comment:
Alternatively, we could do this as a function on the `operator` model, like  `user_has_access`  on `operation` model